micro-rdk/src/common/exec.rs

```diff
@@ -11,13 +11,13 @@ use crate::common::webrtc::exec::WebRtcExecutor;
 
 #[derive(Clone, Debug, Default)]
 /// This executor is local and bounded to the CPU that created it usually you would create it after spwaning a thread on a specific core
-pub struct NativeExecutor {}
+pub struct CPUBoundExecutor {}
```

Comment:
Here I think I'd keep it as `CPUBoundExecutor`, and then re-export it as `Executor`. That way, later, if we have an environment where we had two cores available to micro-rdk (on, say, an ESP32-P4 which has two performance cores and one lesser core), we could swap in another implementation based on features tests or targeting.

But actually, it really isn't pinned to a CPU. It is pinned to a thread. It just so happens that we have pinned that thread to a core.